docs/competition/competition-roles.md

```diff
@@ -1,57 +1,50 @@
 # Competition Roles
 
-To run a successful competition, there are a number of roles which need to be assigned. One role may be filled by multiple volunteers, and one volunteer may be filling multiple roles. Some roles should only be assigned to a single volunteer, and this be their only role.
+To run a successful competition, there are a number of roles which need to be assigned. One role may be filled by multiple volunteers, and one volunteer may be filling multiple roles. Some roles should only be assigned to a single volunteer, and this be their only role. Roles are the responsibility of the given [area](../programme/area-owners.md).
 
 Depending on the nature of the event, some of these roles may not be applicable.
 
 !!! note
     This list may not be complete
 
-## Miscellaneous
-
-- [Intro Briefer](./event/intro-brief.md)
-- [Refreshments](./volunteers/refreshments.md)
-- [Team Pit Manager](./team-pits/README.md)
-- [Robot Regulations Inspector](./team-support/regulations-inspector.md)
-- Group Photo Photographer
-- [Health and Safety Coordinator](./event/incident-management.md)
-
-## Matches
+## Match Operations
 
+- [Tinker Time Coordinator](./team-support/tinker-time.md)
 - [Commentators](./matches/commentating.md)
 - [Judge](./matches/judging.md)
 - [Match Scorer](./matches/match-scoring.md)
-
-### Shepherding
-
 - [Head Shepherd](./matches/shepherding.md#head-shepherd)
 - [Team Shepherds](./matches/shepherding.md#team-shepherds)
 - [Badger](./matches/shepherding.md#badgers)
 - [Roving Shepherd](./matches/shepherding.md#roving-shepherd)
-
-### Arena
-
+- [Intro Briefer](./event/intro-brief.md)
+- Closing Ceremony presenter
+- [Robot Regulations Inspector](./team-support/regulations-inspector.md)
```

Comment:
Think the first two of these should be in Production and the last one in Team Support?